src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.py_in

```diff
@@ -289,7 +327,14 @@ class FitMultipleModel():
             images_per_seg = self.images_per_seg_valid
             self.info_str += "\n\tValidation set after iteration {0}:".format(epoch)
         for mst in self.msts:
-            model_arch, _ = get_model_arch_weights(self.model_arch_table, mst[self.model_id_col])
+            DEBUG.start_timing('eval_query_weights')
+            weights = query_weights(self.model_output_tbl, self.model_weights_col,
```

Comment:
we just need to query the model_arch, don't need to query the weights right?

Reply:
Right, good catch!  Took it out